src/components/TabMenu/Item/style.css

```diff
@@ -22,7 +22,7 @@
 }
 
 .active {
-  color: var(--rvr-gray);
-  font-weight: 700;
+  color: var(--rvr-gray-90);
+  font-weight: 800;
```

Comment:
I just checked in the storybook app - looks like the .active class font-weight is being overridden by the Button font weight.

The `.active` style gets added to an `li` in `TabMenu/Item`, but the `button` element is inside the `li`, and it has it's own font-weight.

I can help you figure out how to get an `.active` class onto the button if that would help.